opensrp-core/build.gradle

```diff
@@ -226,72 +223,57 @@ dependencies {
     }
 
     implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
-    compileOnly 'org.projectlombok:lombok:1.18.12'
-    annotationProcessor 'org.projectlombok:lombok:1.18.12'
+    compileOnly 'org.projectlombok:lombok:1.18.24'
+    annotationProcessor 'org.projectlombok:lombok:1.18.24'
 
-    implementation 'org.smartregister:opensrp-client-utils:0.0.4-SNAPSHOT'
+    implementation 'org.smartregister:opensrp-client-utils:0.0.6-SNAPSHOT'
 
-    implementation 'org.smartregister:opensrp-plan-evaluator:1.6.6-SNAPSHOT'
+    implementation 'org.smartregister:opensrp-plan-evaluator:1.6.7-SNAPSHOT'
 
     implementation 'xerces:xercesImpl:2.12.0'
 
-    implementation('ch.acra:acra:4.5.0') {
-        exclude group: 'org.json', module: 'json'
-    }
-
-    implementation 'com.github.ybq:Android-SpinKit:1.2.0'
-    implementation 'com.mcxiaoke.volley:library:1.0.19'
```

Comment:
Was this removed? Will it be replaced by another library?

Reply:
Yeah for images we are now using the [Glide library](https://github.com/bumptech/glide)